doc/manual/manual.tex

```diff
@@ -9921,6 +9921,42 @@ \subsubsection{Benchmarks for operator splitting}
+
+This section discusses the viscoplastic thermal convection benchmarks described by Tosi et al. \cite{T15}.  The 5 benchmarks extend those of Blankenbach et al. \cite{BBC89} with temperature-, pressure- and strain rate-dependent rheology. As the \aspect{} results are published in the original paper, we limit ourselves to a brief description of the setup and results of the first 2 benchmark cases. 
+
+All 5 benchmarks solve for Boussinesq convection in a box of 1x1 dimensions with free slip boundary conditions. The initial temperature distribution considers a linear depth profile with a slight perturbation to start convection. Top and bottom boundaries are set to a fixed temperature value. The parameters shared between the benchmark cases can be found in \url{benchmarks/tosi_et_al_2015_gcubed/Tosi_base.prm}. The other input files describe the variations on this base model, which pertain to the rheological description. The specific rheologies used are implemented in \url{benchmarks/tosi_et_al_2015_gcubed/tosi.cc} and describe a linear and a plastic component of the viscosity:
+\begin{align}
+  \eta_{lin}(T,z) &= \exp(-\ln(\eta_T) * T + \ln(\eta_Z) * z)
+  \label{eq:tosi-benchmark-lin-visc}
+  \eta_{plast}(\dot\epsilon) &= \eta^* + \frac{\sigma_y}{\sqrt(\dot\epsilon:\dot\epsilon)}
+  \label{eq:tosi-benchmark-plast-visc}
+\end{align}
+where $\eta^*$ is the constant effective viscosity at high stresses and $\sigma_y$ the yield stress.
+
+\paragraph{Case 1: Temparature-dependent convection.}
+\label{sec:benchmark-tosi-case-1}
+
+The first benchmark considers a linear viscosity that only depends on temperature (Eq. \ref{eq:tosi-benchmark-lin-visc}, with $\gamma_Z=0$). When run to steady-state, this produces one convection cell with a high viscosity, stagnant lid insulating the fluid below (see Fig. 1 of \cite{T15}). In \cite{T15}, results of different codes are compared by looking at the average temperature, the Nusselt number at the top and bottom of the domain, the RMS velocity at the top boundary and in the whole domain and the maximum velocity at the surface. These quantities can be queried by using several of \aspect{} postprocessors, but the additional postprocessor in \url{benchmarks/tosi_et_al_2015_gcubed/tosi.cc} is needed to compute the average rate of work done against gravity, the average rate of viscous dissipation and the error between them. Differences between these diagnostic quantities of the 11 codes that participated in the benchmark effort are smaller than 5\% for their preferred mesh resolution.
+
+\paragraph{Case 2: Viscoplastic convection.}
+\label{sec:benchmark-tosi-case-2}
+Case 2 includes a strain rate dependent component in the viscosity, which is harmonically averaged with the linear component (see Eq. \ref{eq:tosi-benchmark-ave-visc} and the code snippet below).
+This rheology leads to mobile-lid convection, with the descending cold lid cooling the cell's interior (Fig. 2 of \cite{T15}).
+
+\begin{equation}
```

Comment:
I would move the formula after line 9947 since you refer to it there, and then you don't need the reference there.